VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -89,7 +91,13 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
 
     ngOnInit(): void {
         this.audioMuted = this.videoCallService.pexipAPI.call.mutedAudio;
-        this.videoMuted = this.videoCallService.pexipAPI.call.mutedVideo || this.videoCallService.isAudioOnlyCall;
+        this.videoMuted = this.videoCallService.pexipAPI.call.mutedVideo || this.audioOnly;
+
+        this.userMediaService.isAudioOnly$.subscribe(audioOnly => {
```

Comment:
does this need to be unsubscribed? I dont see a reference to this on the on destroy lifecycle method

Reply:
Yes, good spot.